examples/github-complete/README.md

```diff
@@ -1,14 +1,29 @@
-# Complete Atlantis example with GitHub Webhooks
+# Complete Atlantis example with GitHub App and Webhooks
 
 Configuration in this directory creates the necessary infrastructure and resources for running Atlantis on Fargate plus GitHub repository webhooks configured to Atlantis URL.
 
 An existing Route53 hosted zone and domain is required to deploy this example.
 
-GitHub's personal access token can be generated at https://github.com/settings/tokens
+The GitHub App can be generated using multiple methods:
+
+- You can follow Atlantis instructions depicted [here](https://www.runatlantis.io/docs/access-credentials.html#github-app). The Atlantis method mostly automates the GitHub App generation using [GitHub App Manifest](https://docs.github.com/en/developers/apps/building-github-apps/creating-a-github-app-from-a-manifest), but you need an exposed endpoint to complete the process.
+- The other method is to manually create the GitHub App as instructed [here](https://docs.github.com/en/developers/apps/building-github-apps/creating-a-github-app).
```

Comment:
I also ran into a wall with this method. The Github docs on setting up a Github App  are very generic and doesn't explain how I should put values in for sections like `Identifying and authorizing users` or `Webhook URL` to work with Atlantis. So I'm left without a path forward to test this out.

Reply:
I understand the confusion. This is an advanced setup which requires the user to be comfortable with GitHub Apps, hence the above recommended path depicted by the Atlantis team.

Let me try and make this clearer:

- You create a GitHub App and give it a name - that name must be unique across the world (you can change it later).
- Provide a valid Homepage URL (this can be the atlantis server url, for instance `https://atlantis.mydomain.com`)
- Provide a valid Webhook URL. The Atlantis webhook server path is located by default at `https://atlantis.mydomain.com/events`
- Generate a Webhook Secret - this is used for Atlantis to trust the deliveries. This is your `github_webhook_secret`.
- Generate a Private Key - this is your `github_app_key`
- On the App's settings page (at the top) you find the App ID. This is your `github_app_id`
- On the **Permissions & Events** you need to setup all the permissions and events according to [Atlantis documentation](https://www.runatlantis.io/docs/access-credentials.html#permissions)

Now you need to Install the App on your organization.

A self-provisioned GitHub App usually has two parts: the App and the Installation.
- The App part is the first step and its where you setup all the requirements, such as authentication, webhook, permissions, etc...
- The Installation part is where you add the created App to an organization/personal-account. It is on the installation page where you setup which repositories the application can access and receive events from.

Reply:
All of this is great info to add to the module docs for users